### PR TITLE
[LUANDA-2020] Add new event.

### PR DESCRIPTION
**Pull request by JovanyNegocio**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
We need to sort out these merge conflicts. I'm out of time for the moment; the best way is if they come up to date with master, but if that's not possible, I'd go with removing/re-adding the redirects file from the PR.

**Comment by bridgetkromhout** [comment]
Hi, @JovanyNegocio - I took a look at your fork of this repo, and I see a problem: "This branch is 13 commits ahead, 643 commits behind devopsdays:master." - please bring your fork up to date, since that will help rule out all sorts of issues with this PR.

Please follow these directions: https://github.com/devopsdays/devopsdays-web/blob/master/CONTRIBUTING.md#editing-your-site - even if you don't want to make a branch, at the very least we need you to pull the upstream changes into your fork:

```
git checkout master
git pull upstream master --rebase
```

**Comment by JovanyNegocio** [comment]
> Hi, @JovanyNegocio - I took a look at your fork of this repo, and I see a problem: "This branch is 13 commits ahead, 643 commits behind devopsdays:master." - please bring your fork up to date, since that will help rule out all sorts of issues with this PR.
> 
> Please follow these directions: https://github.com/devopsdays/devopsdays-web/blob/master/CONTRIBUTING.md#editing-your-site - even if you don't want to make a branch, at the very least we need you to pull the upstream changes into your fork:
> 
> ```
> git checkout master
> git pull upstream master --rebase
> ```

Ok, i'will try it. thanks for help us.

File: data/events/2020-luanda.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+name: "2020-luanda" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2020" # The year of the event. Make sure it is in quotes.
+city: "Luanda" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysluanda" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```
Comment (bridgetkromhout) [inline data/events/2020-luanda.yml]: This does not exist. You will want to register it.

**Comment by bridgetkromhout** [comment]
`BIN +4.3 MB static/events/2020-luanda/organizers/jovanynegocio.jpg`
I recommend replacing this image with a smaller one, like the other two. We don't need to increase the repo size by this much just for a profile picture that will be displayed much smaller.

**Comment by bridgetkromhout** [comment]
Please email info@devopsdays.org with the full names and email addresses of the new team, so we can update Slack and the mailing list.

**Comment by somatorio** [comment]
Olá @JovanyNegocio, estamos pendentes sobre o email para info@devopsdays.org com as pessoas listadas na página de contato e seus respectivos emails (por favor, não coloque os emails aqui nesta discussão pública)

Poderia fazer isso o mais rápido o possível?

File: data/events/2020-luanda.yml (hunk 1 above)
Comment (JovanyNegocio) [reply data/events/2020-luanda.yml]: > `BIN +4.3 MB static/events/2020-luanda/organizers/jovanynegocio.jpg`
> I recommend replacing this image with a smaller one, like the other two. We don't need to increase the repo size by this much just for a profile picture that will be displayed much smaller.

I will change this image now. thanks.

**Comment by JovanyNegocio** [comment]
> Olá @JovanyNegocio, estamos pendentes sobre o email para [info@devopsdays.org](mailto:info@devopsdays.org) com as pessoas listadas na página de contato e seus respectivos emails (por favor, não coloque os emails aqui nesta discussão pública)
> 
> Poderia fazer isso o mais rápido o possível?

Entendido!

**Comment by JovanyNegocio** [comment]
> Olá @JovanyNegocio, estamos pendentes sobre o email para [info@devopsdays.org](mailto:info@devopsdays.org) com as pessoas listadas na página de contato e seus respectivos emails (por favor, não coloque os emails aqui nesta discussão pública)
> 
> Poderia fazer isso o mais rápido o possível?

Done! @somatorio

**Comment by JovanyNegocio** [comment]
> Please email [info@devopsdays.org](mailto:info@devopsdays.org) with the full names and email addresses of the new team, so we can update Slack and the mailing list.

done! @bridgetkromhout

**Pull request merged**